app/main.js

```diff
@@ -1,7 +1,8 @@
 // process.on('uncaughtException', (...args) => console.error(...args))
-const {app, BrowserWindow, Tray, Menu, ipcMain, shell, dialog, globalShortcut} = require('electron')
+const { app, BrowserWindow, Tray, Menu, ipcMain, shell, dialog, globalShortcut } = require('electron')
```

Comment:
so somewhere we use `{ xxx` and somewhere `{xxx`. We should decide which one we wanna have. Are there some reason to have a space there?

Reply:
I guess the same would go for `()`

Reply:
I noticed that for `()` we use no space

Reply:
Must have been something that happened because of space settings in my code editor, I'll fix them back to no spaces.

Reply:
I don't know which one is correct ;] Probably different editores/people use different settings